tests/unit/OrgPerson.spec.ts

```diff
@@ -46,8 +45,7 @@ const validPersonData = {
     middleName: 'D',
     organizationName: '',
     partyType: 'person',
-    email: 'completing-party@example.com',
-    taxId: '0000'
```

Comment:
At the moment, Incorporation/Registration Number and Business Number are not displayed for directors.